### PR TITLE
Bug removed #109

### LINKED ISSUE #109: App Crash if empty file path for Reminder tone
App crashes if the file path is empty in SET REMINDER TONE as shown below -
ezgif com-optimize

### PR DESCRIPTION
**Pull request by codingblazer**
Fixed #109 
App was crashing is user enters a empty or invalid file path. The problem is solved now.

### CODE REVIEW

File: malaria-app-android/src/main/java/com/peacecorps/malaria/activities/RemainderToneActivity.java [hunk 1]
```diff
@@ -71,9 +76,16 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
...
                 path.setText(MP3Path);
-
-
             }
         }
     }
+    public boolean isAudioFile() {
+        if(audioFileUri!=null)
+        {
+            String type=     getContentResolver().getType(audioFileUri);
```
Comment (chhavip) [inline malaria-app-android/src/main/java/com/peacecorps/malaria/activities/RemainderToneActivity.java]: Use proper spaces

File: malaria-app-android/src/main/java/com/peacecorps/malaria/activities/RemainderToneActivity.java [hunk 2]
```diff
@@ -71,9 +76,16 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
...
                 MP3Path = audioFileUri.getPath();
                 path.setText(MP3Path);
-
-
             }
         }
     }
+    public boolean isAudioFile() {
+        if(audioFileUri!=null)
+        {
```
Comment (chhavip) [inline malaria-app-android/src/main/java/com/peacecorps/malaria/activities/RemainderToneActivity.java]: Dont carry opening braces to next line

File: malaria-app-android/src/main/java/com/peacecorps/malaria/activities/RemainderToneActivity.java [hunk 3]
```diff
@@ -51,6 +50,12 @@ public void onClick(View v) {
                 break;
 
             case R.id.dialogButtonOKReminder:
+
+                if("".equals(path.getText()) || !isAudioFile())
```
Comment (chhavip) [inline malaria-app-android/src/main/java/com/peacecorps/malaria/activities/RemainderToneActivity.java]: Use StringUtil.isEmpty function to check empty strings. Do not carry starting brace to next line.

**Comment by codingblazer** [comment]
Hey @chhavip , I have made the requested changes. Please review it.

**Comment by chhavip** [comment]
@codingblazer, looks good! Merging.

**Pull request merged**